site2/docs/client-libraries-go.md

```diff
@@ -302,6 +302,91 @@ fmt.Println(msg.Payload())
 canc()
 ```
 
+#### How to use Prometheus metrics in producer
+
```

Comment:
@imryao thanks for adding docs. 

To allow users better understand the purpose of this part, can you add some explanations/context before the code example? For example, `what does the example do?`, `Are there any considerations that need to take into account when using Prometheus metrics in Go SDK?`, etc.

Reply:
Thanks for your advice! Context and description has been updated~